ValueReference/ReferenceTypesExamples/AgendaService.cs

```diff
@@ -12,67 +12,78 @@ public void ComputeAgenda()
 
             // 1. Create a method that will allow to update an agenda with following values
             // AgendaColor = Black, Name = Black Agenda, NumberOfFiles = 250, Id = {previousValue} * 1000 + {CurrentYear}
+            //Name: SetHardcodedAgenda();
 
             // 2. update the object myAgenda created above using the method created at point 1.
+            myAgenda.SetHardcodedAgenda();
             Console.WriteLine($" My Agenda: Id = {myAgenda.Id}, AgendaColor = {myAgenda.AgendaColor}, Name = {myAgenda.Name}, NumberOfFiles = {myAgenda.NumberOfFiles} ");
 
             // 3. Create a method that will allow to create a copy of an agenda reference.
-
-            AgendaClass referenceAgenda = new AgendaClass();
+            AgendaClass referenceAgenda = myAgenda.CreateSameReferenceAgenda();
```

Comment:
It's nice how you have implemented but for now please create a method in `AgendaService` class that will do this.